src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,56 @@
+package duke.command;
+
+import java.time.format.DateTimeParseException;
+
+import duke.logging.Storage;
+import duke.logging.TaskList;
+import duke.logging.Ui;
+
+/**
+ * The AddCommand class denotes an add command to the Duke chat bot.
+ */
+public class AddCommand extends Command {
+    private final String type;
+
+    /**
+     * Constructs an AddCommand.
+     * @param type              The type of task to be added into the list.
+     * @param taskDescription   The description of the task.
+     */
```

Comment:
Should there be an empty line between description and parameter section for javadocs? I noticed this for your other javadocs as well.